addons/stock/migrations/8.0.1.1/post-migration.py

```diff
@@ -708,6 +716,169 @@ def migrate_procurement_order(cr, registry):
             cr, uid, procurement_ids, {'warehouse_id': warehouse.id})
 
 
+def _move_assign(env, move):
```

Comment:
@pedrobaeza @StefanRijnhart This change helped reduced processing time to 1/2. Can you please review from your end that the logic that has been added looks fine from your end?